### PR TITLE
feat: added docs for big blue button

### PR DESCRIPTION
**Pull request by AhtishamShahid**
Added Docs for Big blue button LTI Provider.

Screen Shot 2022-07-14 at 12 05 35 PM

### CODE REVIEW

File: en_us/shared/course_assets/course_live.rst [hunk 1]
```diff
@@ -82,3 +83,37 @@ Once successfully enabled, Zoom will be accessible in "Live" tab in course navig
...
+the app on `Big Blue button site <https://bigbluebutton.org/teachers/features/>`_.
+
+There are multiple ways to integrate BBB in course, which are discussed below.
+
+1.  If the organization has configured the free tier for the Big blue button, you can select the Free tier and save the settings.
+no further configuration is required.
+
+.. image:: ../../../shared/images/course_live_bbb_free.png
+
+2.  If the Free tier is not configured or you do not want to use free tier, you can deploy BBB on your own server or use third-party hosting providers.
```
Comment (asadazam93) [inline en_us/shared/course_assets/course_live.rst]: I think we should either specify the extra features provided by the commercial tier or provide a link that specifies  the extra features.

Comment (ayub02) [reply en_us/shared/course_assets/course_live.rst]: Agreed. @AhtishamShahid you can use this link: https://bigbluebutton.org/commercial-support/

Comment (AhtishamShahid) [reply en_us/shared/course_assets/course_live.rst]: @ayub02 This link point to a list of 3rd party providers and does not list extra features.

File: en_us/shared/course_assets/course_live.rst [hunk 2]
```diff
@@ -82,3 +83,42 @@ Once successfully enabled, Zoom will be accessible in "Live" tab in course navig
...
+   you can select the Free tier and save the settings. No further configuration is required.
+
+   .. image:: ../../../shared/images/course_live_bbb_free.png
+
+#. If the Free tier is not configured or you do not want to use free tier,
+   you can deploy BBB on your own server or use third-party hosting providers.
+   Please find more information on features available for free tier and commercial
+   version in this `document <https://bigbluebutton.org/free-bigbluebutton-service-for-edx/>`_.
+
+#. You can use the services listed on this page. These services provide complete
```
Comment (asadazam93) [inline en_us/shared/course_assets/course_live.rst]: listed on the link below

File: en_us/shared/course_assets/course_live.rst [hunk 3]
```diff
@@ -82,3 +83,42 @@ Once successfully enabled, Zoom will be accessible in "Live" tab in course navig
...
+Big Blue Button setup
+=====================
+
+`Big blue button <https://bigbluebutton.org/>`_ can be used to integrate video conferencing tool with multiple features.
+
+Instructors can use the big blue button to schedule and start meetings and students can
+see and join these meetings to attend classes. Learn more about
+the app on `Big Blue button site <https://bigbluebutton.org/teachers/features/>`_.
+
+There are multiple ways to integrate BBB in course, which are discussed below.
```
Comment (asadazam93) [inline en_us/shared/course_assets/course_live.rst]: BBB is available in multiple variants, which are discussed below.

**Review by asadazam93 — APPROVED** [review]
Just a couple of comments. The rest looks good

**Pull request merged**